src/index.test.ts

```diff
@@ -63,6 +66,9 @@ describe('public convertAxeToSarif API', () => {
     });
 });
 
+// This initializes global state that the reporter API assumes is available
+require('axe-core');
```

Comment:
Is the place we're making this call relevant? Is it in this line on purpose?

Reply:
It's intentionally near the specific tests that require it, yes (it immediately precedes the tests of the "reporter API" that the comment mentions)